lib/charms/mysql/v0/mysql.py

```diff
@@ -134,7 +134,7 @@ def wait_until_mysql_connection(self) -> None:
 # Increment this major API version when introducing breaking changes
 LIBAPI = 0
 
-LIBPATCH = 71
+LIBPATCH = 72
```

Comment:
I read 72 was planned for @shayancanonical in some of reviews...

Reply:
It's the next one, so it depends who merges it first